zk-token-sdk/src/encryption/auth_encryption.rs

```diff
@@ -30,6 +22,14 @@ use {
     subtle::ConstantTimeEq,
     zeroize::Zeroize,
 };
+#[cfg(not(target_os = "solana"))]
+use {
+    aes_gcm_siv::{
+        aead::{Aead, NewAead},
+        Aes128GcmSiv,
+    },
+    rand::{rngs::OsRng, Rng},
+};
```

Comment:
nit: did cargo fmt move this down? it's up to you, but it can be more consistent to keep this block at the top, same with the other files

Reply:
Hm, yeah it seems like cargo fmt always favors the larger chunk to be on top of the smaller chunk of imports. Alternatively, I can put an extra space between the two import chunks, but it seems like this is less clean. I'll leave it for now and maybe I can update it on a follow-up.